### PR TITLE
Libindicator gtk2

### PR DESCRIPTION
**Pull request by PB-repo**

### CODE REVIEW

File: srcpkgs/libappindicator-gtk2/template [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+build_style=gnu-configure
+configure_args="--with-gtk=2 --disable-tests"
+hostmakedepends="pkg-config glib-devel python"
+makedepends="gtk+-devel glib-devel glib-networking libdbusmenu-glib-devel
+ libdbusmenu-gtk2-devel libindicator-gtk2-devel dbus-glib-devel python-devel
+ python-gobject2-devel pygtk-devel"
+depends="libindicator-gtk2>=${version}_${revision}"
+short_desc="Library to allow applications to export a menu (GTK+2)"
+maintainer="Piotr Balcerowski <piotr@balcerowski.org>"
+license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libappindicator-gtk2/template]: please use spdx identifier

File: srcpkgs/libdbusmenu-gtk2/template [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+configure_args="--disable-dumper --disable-static --disable-tests
+ $(vopt_enable gir vala) $(vopt_enable gir introspection) 
+ HAVE_VALGRIND_FALSE='' --with-gtk=2"
+hostmakedepends="automake gettext-devel glib-devel intltool libtool pkg-config
+ $(vopt_if gir 'gobject-introspection vala-devel')"
+makedepends="gtk+-devel"
+depends="libdbusmenu-glib"
+short_desc="Library for passing menus over DBus (GTK+2)"
+maintainer="Piotr Balcerowski <piotr@balcerowski.org>"
+license="GPL-3, LGPL-2.1, LGPL-3"
```
Comment (maxice8) [inline srcpkgs/libdbusmenu-gtk2/template]: ditto

File: srcpkgs/libindicator-gtk2/template [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+revision=1
+wrksrc="libindicator-${version}"
+build_style=gnu-configure
+configure_args="--with-gtk=2"
+hostmakedepends="pkg-config glib-devel"
+makedepends="garcon gtk+-devel libglib-devel"
+depends="libdbusmenu-gtk2"
+short_desc="Convenient symbols and functions for indicators in the Ayatana project (GTK+2)"
+maintainer="Piotr Balcerowski <piotr@balcerowski.org>"
+license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/libindicator-gtk2/template]: ditto

**Comment by PB-repo** [comment]
I corrected templates, so xlint don't show any warnings.
I'm not familiar with the git so I did a new commit:
https://github.com/PB-repo/void-packages/commit/7214a00817abad96d27956c2df4c645adfa0f92d

**Comment by PB-repo** [comment]
I do not know where it came from "<<<<<<< HEAD ..."
Now libappindicator-gtk is correct.
https://github.com/PB-repo/void-packages/commit/082ed0cc1ff47dd808f15939d4c69f128e26e512

**Comment by PB-repo** [comment]
Now licenses are corrected

**Comment by Gottox** [comment]
For what do we need gtk2 support?

**Pull request closed without merging**